src/image.css.ts

```diff
@@ -27,7 +27,6 @@ export const image = style([
         borderRadius: skinVars.borderRadii.container,
     }),
     {
-        zIndex: 1,
```

Comment:
I don't know why we had this, but removing it didn't break any screenshot test (also tested in webapp)